dcap/types/src/quote3.rs

```diff
@@ -156,6 +156,12 @@ impl<T: AsRef<[u8]>> Quote3<T> {
         &self.report_body
     }
 
+    /// Signature data of the Quote
+    pub fn signature_data(&self) -> SignatureData {
+        SignatureData::try_from(&self.raw_bytes.as_ref()[mem::size_of::<sgx_quote3_t>()..])
+            .expect("Signature data was validated during Quote creation.")
```

Comment:
I need to look at this a bit more deeply (as part of this review) but I would expect that this should be returning a borrow?

Reply:
`CertificationData` (nit: IMO this should be called `PemCertificates`, since that's what it is)

Reply:
I've created #220 to track this, I think it's more than re-naming `CertificationData`. Currently thinking that Pem Chain should be a variant of `CertificationData` instead of the only assumed variant.